api_tests/nodes/views/test_node_links_detail.py

```diff
@@ -125,7 +125,7 @@ def test_cannot_delete_if_registration(self):
             pointer_id,
         )
         res = self.app.delete(url, auth=self.user.auth, expect_errors=True)
-        assert_equal(res.status_code, 403)
+        assert_equal(res.status_code, 404)
```

Comment:
This should probably be a 405 Method Not Allowed, because the GET request at that URL should work, but the DELETE method at the same url shouldn't, yes? Or is it 404ing for another reason?